### PR TITLE
Allow D-Bus activation when there are share methods

### PR DESCRIPTION
**Pull request by Tomin1**
3rd party applications may define share methods which in practice
requires D-Bus auto start. Use `Desktop Entry/Exec` value if there is no
`X-Sailjail/ExecDBus` value.

This will cause applications to open a window when they are not running
but they are introspected, however that is rare to happen for random
applications. Without this, allowing `ExecDBus`, the alternative issue is
that the application can not be brought to foreground at all when it's
running but doesn't have a window open.

### CODE REVIEW

**Review by spiiroin — APPROVED** [review]

**Comment by spiiroin** [comment]
Just noticed that there is no bug ref - is that intentional?

**Comment by Tomin1** [comment]
> Just noticed that there is no bug ref - is that intentional?

No, it wasn't. I'll add it. Thanks for noticing! :+1:

**Comment by Tomin1** [comment]
cc @aroditelev

**Comment by Tomin1** [comment]
Added this to daemon/README.md.

File: daemon/README.md [hunk 1]
```diff
@@ -227,11 +227,13 @@ Autogenerated D-Bus autostart configuration
   - for each application that defines:
         - OrganizationName
         - ApplicationName
-        - ExecDBus
+        - ExecDBus or X-Share-Methods
```
Comment (spiiroin) [inline daemon/README.md:230]: Unless I'm mistaken we prefer to have ExecDBus defined also with X-Share-Methods. And that "or" could make one think that having both is not an option. Maybe "A and/or B" or "A or B  or both" or stte?

Comment (Tomin1) [reply daemon/README.md]: Yeah, I think that makes a lot of sense. I also changed the text below this a bit to clarify the role of ExecDBus.

**Review by spiiroin — APPROVED** [review]

**Review by aroditelev — APPROVED** [review]

**Comment by Tomin1** [comment]
I'll merge this and if we end up improving the D-Bus side for apps before release then let's consider reverting this still.

**Pull request merged**